packages/framework/src/services/kafka/kafka-broker.client.test.ts

```diff
@@ -0,0 +1,89 @@
+import { Kafka } from 'kafkajs';
+import * as sinon from 'sinon';
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { KafkaBrokerClient, KafkaConsumePayload } from './kafka-broker.client';
+
+const test = suite('Kafka Broker Client');
+
+test.before.each(async (context) => {
+  context.sandbox = sinon.createSandbox();
+});
+
+test.after.each((context) => {
+  context.sandbox.restore();
+});
+
+test('should call connect', async (context) => {
+  const onProducerConnectSpy = context.sandbox.spy(context.sandbox.fake());
+  const onConsumerConnectSpy = context.sandbox.spy(context.sandbox.fake());
```

Comment:
`sinon.fake()` can be spied already, so no need to wrap a spy around it.

```suggestion
  const onProducerConnectSpy = context.sandbox.fake();
  const onConsumerConnectSpy = context.sandbox.fake();
```